META-INF/maven/org.cru/aws-sdk/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.cru</groupId>
+        <artifactId>aws-sdk</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>aem-AwsSdkJavaOsgi.core</artifactId>
+    <name>aem-AwsSdkJavaOsgi - Core</name>
+    <description>Core bundle for aem-AwsSdkJavaOsgi</description>
+    <packaging>bundle</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>cru-awssdk-osgi</Bundle-SymbolicName>
+                        <Bundle-Description>This bundle wraps the AwsSdk</Bundle-Description>
+                        <Import-Package>
+
+                        </Import-Package>
+                        <Export-Package>
+                            software.amazon.awssdk.cloudsearch.*,
```

Comment:
We do not only use cloudsearch from this SDK.

Reply:
aah, yes. CloudSearch, CloudSearchDomain, and ssm. Any i'm missing?

Reply:
Auth and region calls at least.

Reply:
I think we get those automatically, see MANIFEST.MF
Also: https://github.com/CruGlobal/aem-cloudsearch/pull/47
And: https://github.com/CruGlobal/cru-aem-commons/pull/317

Reply:
There are no exports that I see in the `MANIFEST.MF` here in the PR.